network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html

```diff
@@ -45,60 +45,87 @@ <h2 class="h4-heading">{% trans "Featured stories" %}</h2>
+        </div>
+        <div class="col-sm-12 col-md-8 pt-3">
+          <hr class="mt-0 mb-4">
+          <h4 class="h5-heading mb-3">
+            <a href="{{highlight.highlight.link_url}}">{{highlight.highlight.title}}</a>
+          </h4>
+          <p>{{highlight.highlight.description}}</p>
+        </div>
       </div>
+      {% endif %} {% endfor %}
     </div>
-    {% endif %} {% endfor %}
   </div>
-</div>
-<div class="row">
-  <div class="col">
-    <hr class="hr-emphasis mb-4">
-  </div>
-</div>
-<div class="row">
-  <div class="col-md-6">
-    <h2 class="h4-heading">{% trans "Trending projects" %}</h2>
+  <div class="row">
+    <div class="col">
+      <hr class="hr-emphasis mb-4">
+    </div>
   </div>
-  <div class="col-md-6 d-none d-md-flex justify-content-end">
-    <a class="cta-link" href="https://www.mozillapulse.org/add">{% trans "Add your project" %}</a>
+  <div class="row">
+    <div class="col-md-6">
+      <h2 class="h4-heading">{% trans "Trending projects" %}</h2>
+    </div>
+    <div class="col-md-6 d-none d-md-flex justify-content-end">
+      <a class="cta-link" href="https://www.mozillapulse.org/add">{% trans "Add your project" %}</a>
+    </div>
   </div>
-</div>
-<div class="row">
-  <div class="col">
-    <div class="pulse-project-list" data-query="" data-max="6" data-featured="True" data-reversed="True"></div>
+  <div class="row">
+    <div class="col">
+      <div class="pulse-project-list" data-query="" data-max="6" data-featured="True" data-reversed="True"></div>
+    </div>
   </div>
-</div>
-<div class="row">
-  <div class="col text-center mb-5">
-    <a class="btn btn-secondary mb-4" href="https://www.mozillapulse.org">{% trans "More from our network" %}</a>
-    <a class="cta-link d-block d-md-none" href="https://www.mozillapulse.org/add">{% trans "Add your project" %}</a>
+  <div class="row">
+    <div class="col text-center mb-5">
+      <a class="btn btn-secondary mb-4" href="https://www.mozillapulse.org">{% trans "More from our network" %}</a>
+      <a class="cta-link d-block d-md-none" href="https://www.mozillapulse.org/add">{% trans "Add your project" %}</a>
+    </div>
   </div>
-</div>
-<div class="row">
-  <div class="col">
-    <hr class="hr-emphasis mb-5">
-    <div class="row mb-5">
-      <div class="col">
-        <div class="feature-quote">
-          <h3>{% trans "We want to give users the knowledge and the power to make informed decisions by themselves to prevent abusive practices." %}</h3>
-          <p class="h6-heading">{% trans "Rishab Nithyanand, Mozilla Fellow" %}</p>
-        </div>
-      </div>
+  <div class="row section-quote">
+    {% with quote_image=page.quote_image %}
+    {% if quote_image %}
+    <div class="col-md-4 col-lg-6 mb-5 mb-md-0">
+      {% comment %}
+        To-do:
+        1) Update picture_ratios.html so image sizes are passed as arguments instead of hard-corded.
+        2) Replace the following "picture" code with {% optimize_images page.quote_image %}
```

Comment:
@mmmavis Does this todo need to be addressed right now? If not, can we make sure there's a ticket open for this, please?

Reply:
This needs to wait until we are done updating the new hero section. I will file a follow-up ticket!

Reply:
Filed: https://github.com/mozilla/foundation.mozilla.org/issues/4924